File: java/ooad/TicTacToeApp/src/com/techlabs/tictactoe/Player.java
package com.techlabs.tictactoe;

public class Player {
	private String name;
	private MarkType mark;
	
	public Player(String name,MarkType mark){
		this.name=name;
		this.mark=mark;
	}
	
	public String getName(){
		return this.name;
	}
	
	public MarkType getMark(){
		return this.mark;
	}
	
	@Override
	public String toString(){ 
		return this.name+" -> "+this.mark.toString();
	}
}

File: dotNet/ClassLibrary/WinForm/TicTacToeApp/TicTacToeCore/Player.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TicTacToeCore
{
    public class Player
    {
        private string _name;
        private MarkType _mark;

        public Player(String name,MarkType mark)
        {
            _name = name;
            _mark = mark;
        }

        public String Name
        {
            get
            {
                return _name;
            }
        }

        public MarkType Mark
        {
            get
            {
                return _mark;
            }
        }

        public override string ToString()
        {
            return Name+" -> "+Mark.Name;
        }
    }
}
